### ISSUE TITLE
"/bin/sh: 1: git: not found" when running docker image

### ISSUE THREAD

**Issue by mmanhertz** [WPO-Foundation/wptagent#73]
Hi, 
when I run the docker image (either locally on windows or in our rancher environment) I get the following output

```
/bin/sh: 1: git: not found
Python 2.7.12
Modern traceroute for Linux, version 2.0.21
Copyright (c) 2008  Dmitry Butskoy,   License: GPL v2 or any later
chmod: cannot access '/root/.config/configstore/': No such file or directory
RTNETLINK answers: Operation not permitted
Cannot find device "ifb0"
RTNETLINK answers: Operation not permitted
Cannot find device "ifb0"
bad action parsing
parse_action: bad value (5:mirred)!
Illegal "action"
Cannot find device "ifb0"
13:26:33.368 - Get Work Error: None
13:26:33.479 - Get Work Error: None
13:26:33.587 - Get Work Error: None
...
```

Am I missing something in the setup? I followed the `docker.md` for the local run and adjusted as needed for rancher. To me this seems to be either incomplete documentation or a broken DockerFile. (Though I wouldn't rule out human error on my side either ;) )

**Comment by pmeenan** [comment]
The missing git command shouldn't be a problem (though worth updating the docker image to install).  It looks like a couple of things are going on:

1 - It is trying to initialize traffic-shaping but the interfaces aren't available (shaping won't work inside of docker on Windows)
2 - It's failing in a tight loop to poll for work (server not configured?)

It kind of looks like the docker-specific environment variables for config aren't being passed through.

@sburnicki is the expert on running the agent in a docker environment.

**Comment by mmanhertz** [comment]
Hmm... ok, I was assuming the first error was causing the others.

I've also tried running with `-e "SHAPER=none"` and you are right, the `RTNETLINK` errors go away.

I reviewed everything else I did with regards to server configuration and stumbled across this note from the docker.md that I had previously overlooked:
`SERVER_URL: will be passed as --server (note: it must end with '/work/')`

The agent can now successfully connect to our server. Thanks for the help. 

Technically the `"/bin/sh: 1: git: not found"` issue is still valid, so I'll leave this open. :)
